src/main/java/edu/wpi/grip/core/operations/composite/BlurOperation.java

```diff
@@ -48,7 +48,7 @@ public String getName() {
 
     @Override
     public String getDescription() {
-        return "Soften the details of an image to remove noise";
+        return "Soften the details of a Mat to remove noise.";
```

Comment:
I would leave this as-is.  It's much clearer to the user to say "image" than "Mat".